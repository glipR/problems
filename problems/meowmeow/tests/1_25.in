100
buuuuuuuuuuuuuuurrrrrrrrrrrrrrrrrrrbrrblllle
hdgmvrosotril
donjzijxkkoe
uvuzkssgpknbgg
ht
bn
qqbztriiiiiiiciiiiiiiillllllllllllllllllllll
hisssssssssssssssssssssssssssssssssssssssh
cgqaagrj
kwbbvzwklxwzixqxs
hss
triiiiiiiiiiiiiidiiiiiiiiiiiilllllll
hss
wtriiillllll
hissssssssssssssssssssss
gyaqaua
hisssssssssssssssssssssssssk
mnajykmfhhnzqarrflwuxfqt
hisssssssss
bamknzdy
hisssssssssssssssssssssssss
mhcdtpjq
hisssssssssssssssssssssssssssssssssssssssss
tril
trll
hfgkukgbtktriiiiiiiiiiiieiiillllll
hissssssssssssssssssssssssssssssssssssssssssssssssk
hissssssss
unptvifhkcvx
hisssssssssssssssssssssssssss
brblle
hissssssss
triiiilllllllllllll
buuuuuuuuuucrrrrrrrrrrrrrrrrrrrrrrrblllllllllle
hss
vxbuhwokfbon
fntrwkpbuuuuuuuuuuuuuuuuuuuuuuurrrrrrrrrrrrrrrrrrrrrrrblllllllllllllllle
iyozxkoymuybuwb
buuuuuuuuuuuuuuuuuuuurrrrrrrrrrrbllllle
jafvrxgjshtm
hisssssssssssssssssssssssssssssssssssssssss
buuuuuuuujuuuuuuuuuuuuuuuuuurrrrrrbllllllllllllllllllllllle
svwlctriiiiiiiiiiiiiiiiiisiiiiiiiiilllllllllllllllllllllllllll
triiiiiiiiiiiiiiiiiillllllllllllllllllllllllllllllo
hss
buuuuuuuuuuuuuuuuuuuurrrrrrrrrrrrrrrrrrrblllllllllllllllllile
triiiiiiiiiiiiiillllll
buuuuuuuuuuuuuuuuuuuuuuuuurrrrrrrrrrrrrrrrrrrrrrrrrrblllllllllllllllllllle
hissssssssssssssssssssssss
htmtdctujijf
ayswijkbk
bswqd
ajmudxspae
pdhisssssssssssssssssssssssss
triiiiiiiiiiiiiiiiiiiillllllllllllllllllllllv
tril
sbuuuuuuuuuuuuuuuuuuurcrrrrrrrrrrble
hss
eankjk
hissssssssssssssssssssssssssssss
bugrrrrrrrrrrrbllllle
hissssssssssfssssssss
deb
vyajeb
qgxafgcyburbe
triiiiiiiiiiiiiiiiiiiiiiilllllllllllllllll
triiiiiiiiiiiiillllllll
hz
hissssssssssss
triiiiiiiiiiiiiiiiiiiiiiiiiiiiiilllll
ervlqvkjqws
yxywylht
mdoosatriiiiiiiiiiiiiiiiiiiiiiiiiiiiillllllllllllllllllllllll
hisssssssssssssssssssssssssssssssssssssssssssss
dpgbnebpd
triiiiiiiiiiiiiiiiiiiiiiiillllllllllllllllllj
triiiiiiiiiiiiikiiiiilllllllllllllllllllll
bftnzuxrbfynfjxdq
gefoiz
hisssssssssssssssswssssssssssssssss
tril
triiiiiiillllllllllllllll
hisssssssssssssssssssssssssssssssssssss
jcnydeefrxlba
hisssssssss
hss
tril
tril
ahhbmdznooiqfjmxxtv
buuuuuuuuuuuuurrrrrrrrrrrrrrrrrrrrrrrrblllllllllllllllllllle
triiiiill
his
pfruz
nhtwkfhss
vjdbytugxtriiiiiiiiiiiiiiiiiiiilllllll
hissssssssssssss
tril
ihavuhss
atriiiiiiiiiiiiiiiiiiiiiiiiiiiillllllllllllllllll
his