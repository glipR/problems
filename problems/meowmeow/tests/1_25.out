human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
hiss
human noises
human noises
human noises
hiss
human noises
hiss
human noises
hiss
human noises
human noises
human noises
human noises
hiss
human noises
hiss
human noises
hiss
trill
human noises
human noises
human noises
human noises
human noises
burble
human noises
hiss
human noises
human noises
human noises
human noises
human noises
trill
burble
hiss
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
hiss
human noises
human noises
human noises
human noises
human noises
trill
trill
human noises
hiss
trill
human noises
human noises
human noises
hiss
human noises
human noises
human noises
human noises
human noises
human noises
human noises
trill
hiss
human noises
hiss
human noises
human noises
human noises
human noises
burble
trill
human noises
human noises
human noises
human noises
hiss
human noises
human noises
human noises
human noises
